deployments/rds-s3/terraform/rds-s3-components/main.tf

```diff
@@ -345,14 +347,14 @@ module "kubeflow_aws_telemetry" {
   depends_on = [module.kubeflow_training_operator]
 }
 
-module "kubeflow_user_namespace" {
-  source            = "../../../../iaac/terraform/common/user-namespace"
-  helm_config = {
-    chart = "${var.kf_helm_repo_path}/charts/common/user-namespace"
-  }  
-  addon_context = var.addon_context
-  depends_on = [module.kubeflow_aws_telemetry]
-}
+# module "kubeflow_user_namespace" {
```

Comment:
Should this be disabled with a flag rather than commenting it out?  Many of the changes you've made could be upstreamed but I don't think this one would be okay to upstream without the ability to include it.

Reply:
Added as a flag in 377b95234b0470dc7e5c6a7736268b2bdb0528e0